indico/modules/events/abstracts/templates/display/submit_abstract.html

```diff
@@ -0,0 +1,28 @@
+{% from 'events/display/indico/_common.html' import render_users %}
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title -%}
+    {% trans %}Submit abstract{% endtrans %}
+{%- endblock %}
+
+{%- block content %}
+    {% if cfa.is_open %}
```

Comment:
That would work too, but with a separate template you can just render it if the initial `if` fails and save one level of indentation of the whole form handling part. Just feels cleaner to me that way...

(of course you could also render the same template up there instead of using a separate one)